src/CoreBundle/EventListener/ClearRoutingCacheListener.php

```diff
@@ -29,7 +29,7 @@ public function __construct(ChameleonBaseRouter $router)
         $this->router = $router;
     }
 
-    public function clearRoutingCache(Event $event)
+    public function clearRoutingCache(RoutingConfigChangedEvent $event)
```

Comment:
The base Event class is used intentionally - we do not care about the concrete event as we don't need any information, so we make this listener compatible to as many events as possible.

Reply:
ok